nano/core_test/socket.cpp

```diff
@@ -57,15 +55,15 @@ TEST (socket, max_connections)
 	client3->async_connect (dst_endpoint, connect_handler);
 
 	auto get_tcp_accept_failures = [&node] () {
-		return node->stats.count (nano::stat::type::tcp, nano::stat::detail::accept_failure, nano::stat::dir::in);
+		return node->stats.count (nano::stat::type::tcp_listener, nano::stat::detail::accept_failure, nano::stat::dir::in);
 	};
 
 	auto get_tcp_accept_successes = [&node] () {
-		return node->stats.count (nano::stat::type::tcp, nano::stat::detail::accept_success, nano::stat::dir::in);
+		return node->stats.count (nano::stat::type::tcp_listener, nano::stat::detail::accept_success, nano::stat::dir::in);
 	};
 
-	ASSERT_TIMELY_EQ (5s, get_tcp_accept_failures (), 1);
-	ASSERT_TIMELY_EQ (5s, get_tcp_accept_successes (), 2);
+	ASSERT_TIMELY_EQ (10s, get_tcp_accept_successes (), 2);
```

Comment:
Minor, do these need 10s or would 5s, which is more standard, suffice?

Reply:
There was some variation in time it took for sockets to become dead, so I decided to increase this timeout to make it reliably pass.